content/concepts/the-rippled-server/transaction-censorship-detection.md

```diff
@@ -50,7 +52,7 @@ Here are some scenarios that could cause the detector to issue false positive me
 
 - Your `rippled` server is out of sync with the network and has not yet realized it.
 
-- Your `rippled` server is being impacted by a class of bug that causes `rippled` servers to inconsistently relay transactions to other `rippled` servers in the network.
+- `rippled` servers in the network, including possibly your own server, are being impacted by a class of bug that causes `rippled` servers to inconsistently relay transactions to other `rippled` servers in the network.
```

Comment:
Last nit: Maybe pitch the bug bounty again here? Or at least encourage people to report the bug if they're seeing this?

Reply:
@mDuo13 - fantastic addition. Just added it... 🙌